.github/workflows/actions/auto-request-reviews/main.js

```diff
@@ -0,0 +1,45 @@
+const core = require('@actions/core');
```

Comment:
Is there a way to write these in TypeScript? I'm admittedly very inexperienced in writing actions, but from a maintainability standpoint, I'm hesitant for us to keep this as JS if we can do it in TS